client/web/src/enterprise/app/settings/AppSettingsArea.tsx

```diff
@@ -61,6 +65,19 @@ const AppSettingsLayout: FC = () => {
                         </Button>
                     </li>
                 ))}
+                <li>
+                    <MenuDivider />
+                </li>
+                <li>
+                    <Button
+                        as={Link}
```

Comment:
We have a component in the Wildcard library for that:

```suggestion
                    <ButtonLink
```

Reply:
I followed the code style surrounding this change. Maybe a separate CL to clean them all?